packages/product-components/src/components/SelectAssetModal/SelectAssetModal.tsx

```diff
@@ -1,137 +1,184 @@
-                    <Button
-                        size="tiny"
-                        variant="tertiary"
-                        textWrap={false}
-                        onClick={() => {
-                            setSearch('');
-                        }}
-                    >
-                        All networks (5)
-                    </Button>
-                    {tokens.map(token => (
-                        <Button
-                            size="tiny"
-                            variant={search === token ? 'primary' : 'tertiary'}
-                            onClick={() => {
-                                setSearch(token);
-                            }}
-                        >
-                            <Row gap={spacings.xs}>
-                                <AssetLogo size={20} coingeckoId={token} placeholder="btc" />
-                                {token}
-                            </Row>
-                        </Button>
-                    ))}
-                </Row>
+                <NetworkTabs
+                    networkCategories={networkCategories}
+                    networkCount={networkCount}
+                    searchCategory={searchCategory}
+                    setSearchCategory={setSearchCategory}
+                />
 
                 {filteredData.length === 0 ? (
-                    <Column alignItems="center" justifyContent="center" height={LIST_HEIGHT}>
-                        <Text variant="tertiary">no data</Text>
-                    </Column>
+                    <AssetItemNotFound
+                        listHeight={LIST_HEIGHT}
+                        searchCategory={searchCategory}
+                        networkCategories={networkCategories}
+                    />
                 ) : (
                     <ShadowContainer>
                         <ShadowTop backgroundColor={shadowColor} />
                         <VirtualizedList
                             items={filteredData}
                             ref={scrollElementRef}
                             onScroll={onScroll}
-                            renderItem={({ name, symbol, isFavorite, badge }) => (
+                            renderItem={({
+                                cryptoId,
+                                name,
+                                symbol,
+                                isFavorite,
+                                badge,
+                                coingeckoId,
+                                contractAddress,
+                            }: AssetProps) => (
                                 <AssetItem
                                     key={`${symbol}-${name}`}
+                                    cryptoId={cryptoId}
                                     name={name}
                                     symbol={symbol}
                                     isFavorite={isFavorite}
                                     badge={badge}
+                                    logo={{
```

Comment:
nit: thinking that it would be more clear this way:

```
logo={<AssetLogo ... />}

Reply:
Done